### ISSUE TITLE
`+unlint` fails to warn when given invalid values, leading to silent suppression failures

### ISSUE THREAD

**Issue by VasilevNStas** [objectionary/lints#1388]
## Problem

Malformed `+unlint` values such as `+unlint foo:abc` or `+unlint foo:` (empty line part) are silently ignored: the lint neither suppresses anything nor warns about the bad format, so a user typo quietly disables an intended suppression.

## Root cause

`src/main/java/org/eolang/lints/LtUnlint.java:62-72` parses the `+unlint` tail into three buckets (`global`, `single`/`range` via regex, and `granular`):

```java
final boolean global = !unlint.matches(UNLINT_LINE_REGEX);
...
final boolean granular = ... starts-with(tail, name + ":") ...
```

A tail like `foo:abc` or `foo:` matches the `granular` predicate but none of the three regex branches, so it falls through with no effect. `LtIncorrectUnlint` validates only the *name* part (up to `:`), not the format after it.

## Minimal example

```eo
+package x
+architect yegor@256.com
+version 0.0.1
+home https://github.com/objectionary
+spdx SPDX-FileCopyrightText: Copyright (c) 2026 Objectionary.com
+spdx SPDX-License-Identifier: MIT
+unlint ascii-only:abc

[] > main
  # Привет
  42 > @
```

The `ascii-only` defect is NOT suppressed (fine), but the user is never told that `ascii-only:abc` is an invalid format — the suppression silently does nothing.

## Expected behavior

A `+unlint` with an invalid line/range part should produce a warning (either from `LtUnlint` or from `LtIncorrectUnlint`), so the user learns about the typo.

## Related

- `LtIncorrectUnlint.java` — currently validates only the lint name.
- The `+unlint` format is parsed independently in three places (`LtUnlint`, `DefectMissing`, `UnlintInRange`) — a shared parser would make validation consistent.

**Comment by 0crat** [comment]
@vasilevnstas Thanks for reporting a new bug! You've earned +12 points. By reporting bugs, you help our project improve its quality. If you find anything else in the repository that [doesn't look](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) as good as you might expect, [do not hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to [report](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) it. Your running score is [+878](https://www.eolang.org/zerocracy/objectionary-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too).

**Comment by opusstudiohq-max** [comment]
This looks already fixed on master. `LtIncorrectUnlint.invalid(tail, name)` checks the spelling after the colon, not only the name:

```java
return !tail.equals(name)
    && !tail.matches(String.format("%s:\d+", name))
    && !tail.matches(String.format("%s:\d+-\d+", name));
```

and `defects()` calls it as `!this.names.contains(name) || invalid(tail, name)`, so `+unlint foo:abc` and `+unlint foo:` are reported with "has an invalid format, use ...". It arrived in 596288d, and `packs/single/incorrect-unlint/catches-invalid-format.yaml` covers it.

The issue text cites the state before that commit. Raising it only because the issue is still open.

---
_Generated by [Claude Code](https://claude.com/claude-code)_
